packages/abstract-core/src/native/version_control.rs

```diff
@@ -48,7 +58,9 @@ pub struct AccountBase {
 }
 
 #[cosmwasm_schema::cw_serde]
-pub struct InstantiateMsg {}
+pub struct InstantiateMsg {
+    pub config: Config,
```

Comment:
Lets accept the config params here, that makes it easier for us to write tests and is just clearer when using it.